package/contents/ui/main.qml

```diff
@@ -49,14 +52,16 @@ Item {
 		readonly property bool active: !!command
 		readonly property bool waitForCompletion: plasmoid.configuration.waitForCompletion
 		readonly property int interval: Math.max(0, plasmoid.configuration.interval)
-		readonly property string command: plasmoid.configuration.command || 'sleep 2 && echo "Test: $(date +%s)"'
+		readonly property string command: plasmoid.configuration.command || ''
```

Comment:
Using `sleep 2 && echo "Test: $(date +%s)"` as a hardcoded default probably came from before I implemented the config for this widget.

Since there isn't a `placeholderText` informing the user of this default command when the TextField is empty then yes we should probably remove it.

Reply:
That said I'll probably need to check that nothing breaks when it tries to run an empty command. After all there is code to "wait for the command to complete".